feathr_project/feathr/spark_provider/local_spark_utils/gen_template.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+spark-submit \
+    --master local[*] \
+    --name $jobName \
+    --packages $packages \
+    --conf "spark.driver.extraClassPath=../target/scala-2.12/classes:jars/config-1.3.4.jar:jars/jackson-dataformat-hocon-1.1.0.jar:jars/jackson-module-caseclass_2.12-1.1.1.jar:jars/mvel2-2.2.8.Final.jar:jars/fastutil-8.1.1.jar" \
```

Comment:
Should we use Jinja to manage the templates, rather than using a plain shell?

Reply:
Not familiar with Jinja, seems a good tool for complex Templates. I could try to adopt it if it's a recommended way.